draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -408,13 +408,13 @@ The cbor representation of a consistency proof for RFC9162_SHA256 is:
 ~~~~ cddl
 consistency-proof =  bstr .cbor [
 
```

Comment:
I also believe it *should* be possible for a consistency proof to show consistency between two arbitrary points in the log. So the 'older' / 'newer' frazing seems like an improvement to me also